tests/e2e/features/3DViewer.feature

```diff
@@ -5,10 +5,11 @@ Feature: preview 3D model
 
 
     Scenario: preview 3D model
-        Given the user has logged in with username "admin" and password "admin"
-        And the user has uploaded the following 3D models:
+        Given the following 3D models have been uploaded:
```

Comment:
maybe filename should be 3DViewerPreview.feature which give us idea what kind of feature we are testing rather than extension name

Reply:
good point, although I think one is not allowed to start file name with a number... "preview3DViewer.feature" should work